package.json

```diff
@@ -4,14 +4,17 @@
   "author": "'Julian Viereck' <julian.viereck@gmail.com>",
   "license": "BSD-2-Clause",
   "main": "./parser",
+  "types": "./parser.d.ts",
   "bin": "bin/parser",
   "homepage": "https://github.com/jviereck/regjsparser",
   "repository": {
     "type": "git",
     "url": "git@github.com:jviereck/regjsparser.git"
   },
   "scripts": {
-    "test": "node test/index.js"
+    "test": "run-p test:*",
```

Comment:
Where does the `run-p` come from - is this some `package.json` syntax?

Reply:
That's a command from the `npm-run-all` package I added